ocs_ci/ocs/utils.py

```diff
@@ -624,3 +627,25 @@ def apply_oc_resource(
     log.info(f"Applying rook resource from {template_name}")
     occli = OCP()
     occli.apply(cfg_file)
+
+
+def collect_ocs_logs(dir_name):
+    """
+    Collects OCS logs
+
+    Args:
+        dir_name (str): directory name to store OCS logs. Logs will be stored
+        in dir_name suffix with _ocs_logs.
+
+    """
+    log_dir_path = os.path.join(
+        ocsci_config.RUN['log_dir'],
+        f"failed_testcase_ocs_logs_{ocsci_config.RUN['run_id']}"
+    )
+    create_directory_path(log_dir_path)
+    dir_name = f"{dir_name}_ocs_logs"
+    dump_dir = os.path.join(log_dir_path, dir_name)
+    cmd = f"adm must-gather --image=ashishranjan738/ocs-must-gather --dest-dir={dump_dir}"
+    log.info(f"OCS logs are placed in location {dump_dir}")
+    occli = OCP(namespace=defaults.ROOK_CLUSTER_NAMESPACE)
```

Comment:
please use namespace from config ENV_DATA section

Reply:
> please use namespace from config ENV_DATA section

ahh,, Its strange I have removed namespace parameters in current commit , its reflecting the same in my branch but not in PR